lib/src/typewriter.dart

```diff
@@ -15,11 +15,17 @@ class TypewriterAnimatedText extends AnimatedText {
   /// By default it is set to 30 milliseconds.
   final Duration speed;
 
+  /// The [Curve] of the rate of change of animation over time.
+  ///
+  /// By default it is set to Curves.ease.
+  final Curve curve;
+
   TypewriterAnimatedText(
     String text, {
     TextAlign textAlign = TextAlign.start,
     @required TextStyle textStyle,
     this.speed = const Duration(milliseconds: 30),
+    this.curve = Curves.ease,
```

Comment:
Same issue with `Typer`...  Is `ease` the _best_ default to be _consistent_ with current functionality?  Or would `linear` be better?

Reply:
Sure, that would seem nice. I shall set the default `curve` to `linear`.

I checked which is the default `curve` in `CurveTween`. Turns out it is `Curves.ease`: ![CurveTween](https://api.flutter.dev/flutter/animation/CurveTween-class.html).